paper.tex

```diff
@@ -35,6 +35,8 @@
 
 \usepackage{longtable}
 
+\hyphenation{Math-e-ma-ti-ca pol-y-morph-ism}
```

Comment:
I would worry about overflows once the article is written, as you mentioned in #117.